io/src/main/scala/laika/render/epub/ZipWriter.scala

```diff
@@ -37,19 +35,18 @@ object ZipWriter {
     * file (called `mimeType`) is written uncompressed. Hence this is not
     * a generic zip utility as the method name suggests.
     */
-  def zipEPUB[F[_]: Sync] (inputFs: Seq[BinaryInput[F]], output: BinaryOutput[F]): F[Unit] = {
+  def zipEPUB[F[_]: Sync] (inputs: Seq[BinaryInput[F]], output: BinaryOutput[F]): F[Unit] = {
```

Comment:
I wonder if some of this `Zip` stuff could be upstreamed into FS2. I think it came up on the Discord the other day.

Reply:
That would be an interesting API design challenge, given how much metadata you'd need to route through such a stream. Also note this funny requirement that EPUB has: the first entry needs to be uncompressed. The API would need to support these kind of edge cases.